packages/host/app/lib/current-run.ts

```diff
@@ -505,6 +505,7 @@ export class CurrentRun {
           identityContext,
         },
       );
+      await api.flushLogs();
```

Comment:
make sure to await any async in the API so that the computeds can be calculated before inserting into the index